register_confirm.php

```diff
@@ -0,0 +1,46 @@
+<?php
+require "header.php";
+?>
+<?php
+$question = $_POST['question'];
+$answer = $_POST['answer'];
+?>
+<?php
+if($question == null || $answer == null){
+    header('Location: http://localhost/FirstPhp/register.php');
+    print "問題と答えを入力してください。";
+    exit();
+} else if (strlen($_POST['question']) > 500 || strlen($_POST['answer']) > 200) {
+    header('Location: http://localhost/FirstPhp/register.php');
+    print "問題は500文字以内、答えは200文字以内で入力してください。";
+    exit();
+} else {?>
+	<form action= "register_confirm.php" method="post">
+    <label for="question">問題</label>
+    <input type="text" name="question"  value="<?= $question ?>"><br />
+    <label for="answer">答え</label>
+    <input type="text" name="answer"  value="<?= $answer ?>"><br />
+    <button type= "submit" name= "confirm">確認</button>
+    </form>
+<?php
+}
+?>
+<?php
+//「確認ボタン」が押下されたらDB登録の処理
+if(isset($_POST['confirm'])){
+$sql = "INSERT INTO questions VALUES(:question)";
+$stmt = $db->prepare($sql);
+$stmt->execute([':question' => $question]);
+
+$sql = "SELECT max(id) as max_id from questions";
```

Comment:
```suggestion
$sql = "select id from questions order by created_at desc limit 1";
```
最新のIDを取得するのはこっちのが良いです。
maxで取得だとdeleteなどで途中のIDを消したりしたらズレると思われるので。